.github/workflows/tag.yml

```diff
@@ -61,33 +55,20 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
+          # DEPLOY_KEY is used by lerna_version step to commit and push version bumps to the repo.
           ssh-key: ${{ secrets.DEPLOY_KEY }}
           # Lerna needs *all* the git history to determine which packages have changed
           fetch-depth: 0
 
+        # "prepare" hook on `bun install` runs `bun run build`
       - name: Setup environment
         uses: ./.github/actions/setup-env
 
-      - name: Bump package versions
+      - name: Bump package versions, commit, tag, and push to Git
         id: lerna_version
         uses: ./.github/actions/lerna-version
         with:
-          update_type: ${{ github.event.inputs.update_type }}
-          prerelease_identifier: ${{ github.event.inputs.prerelease_identifier }}
-
-      - name: Setup Git CLI
-        uses: ./.github/actions/setup-git
-        with:
-          name: ${{ github.actor }}
-
-      - name: Commit and tag v${{ steps.lerna_version.outputs.VERSION }}
-        run: |
-          git add .
-          git commit -m "chore: version bump to ${{ steps.lerna_version.outputs.VERSION }}"
-          # Create an annotated tag (`-m`) so that lerna can use it to determine which packages have changed
-          # See https://github.com/lerna/lerna/issues/1357#issuecomment-438162152
-          git tag "v${{ steps.lerna_version.outputs.VERSION }}" -m "v${{ steps.lerna_version.outputs.VERSION }}"
-          git push && git push --tags
```

Comment:
These git setup, committing, and pushing steps were [moved](https://github.com/canonical/ds25/blob/4bec07bbd7c82333a56162dd91e7ee362641421e/.github/actions/lerna-version/action.yml#L19-L38) into the `lerna-version` action.